lib/galaxy/schema/schema.py

```diff
@@ -2577,6 +2577,32 @@ class Config:
         extra = Extra.allow  # Allow any other extra fields
 
 
+class AsyncTaskResultSummary(BaseModel):
+    id: str = Field(
+        ...,
+        title="ID",
+        description="Celery AsyncResult ID for this task",
+    )
+    ignored: bool = Field(
+        ...,
+        title="Ignored",
+        description="Indicated whether the Celery AsyncResult will be available for retrivial",
+    )
+    name: Optional[str] = Field(
+        ...,
```

Comment:
I think leaving the ellipsis as the default value will make the field required even if it's marked as Optional. Same for the `queue` field below.
```suggestion
        None,
```

Reply:
Okay - I think I've fixed it. Thanks for the catch!